classes/logging.php

```diff
@@ -16,24 +16,30 @@ class Object_Sync_Sf_Logging extends WP_Logging {
 
 	protected $wpdb;
 	protected $version;
+	protected $slug;
+	protected $option_prefix;
 
 	public $enabled;
 	public $statuses_to_log;
 
 
 	/**
-	 * Constructor which sets content type and pruning
+	 * Constructor which sets content type and pruning for logs
 	 *
 	 * @param object $wpdb An instance of the wpdb class.
 	 * @param string $version The version of this plugin.
+	 * @param string $slug The plugin slug
+	 * @param string $option_prefix The plugin's option prefix
 	 * @throws \Exception
 	 */
-	public function __construct( $wpdb, $version ) {
-		$this->wpdb    = $wpdb;
-		$this->version = $version;
+	public function __construct( $wpdb, $version, $slug, $option_prefix ) {
```

Comment:
Since `__construct` is a public method and this is the 1.4 release (not a 2.0 backwards compatibility breaking release), the new parameters should be optional. We can't guarantee developers haven't invoked this method.

Here it's a more likely issue. Developers who want additional content to appear alongside the existing logs will need to have constructed a logging object. From PHP 7.1 onward, this is a fatal error.